canonical-websites/www.ubuntu.com
# Add some missing legal redirects

## Done
Add some missing legal redirects. Went through all the previous legal pages and found some missing redirects.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Check the redirects in the PR work as expected

## Issue / Card
RT: #115292
